tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -64,6 +67,27 @@ def create(self, request, *args, **kwargs):
                 username=settings.ACFTITAN_USERNAME,
                 port=22
             )
+
+            user = request.user
+            data_file = DataFile.objects.get(id=response.data.get('id'))
+
+            # Send email to user to notify them of the file upload status
+            email_context = {
+                'stt_name': str(data_file.stt),
+                'submission_date': data_file.created_at,
+                'submitted_by': user.get_full_name(),
+                'fiscal_year': data_file.fiscal_year
+            }
+
+            recipients = User.objects.filter(
+                location_id=data_file.stt.id,
+                account_approval_status=AccountApprovalStatusChoices.APPROVED,
+                groups=Group.objects.get(name='Data Analyst')
```

Comment:
@jtimpe are only data analysts getting emails? would it be easy to change such that any group that has permission to submit data will get email? like ofa admins for example.

Reply:
edit: im realizing the ticket title is focused on data analysts so we can address in follow-on if easiest.